frescobaldi_app/appinfo.py

```diff
@@ -35,6 +35,7 @@
 maintainer_email = "info@frescobaldi.org"
```

Comment:
Should we change this email address to `frescobaldi@googlegroups.com`? Don't know if you have to be subscribed.